pkg/datapath/tables/route.go

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package tables
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/hashicorp/go-memdb"
+	"golang.org/x/sys/unix"
+
+	"github.com/cilium/cilium/pkg/statedb"
+)
+
+type Route struct {
```

Comment:
I'm trying to avoid exposing netlink here as it's becoming more and more likely we'll switch away from the unmaintained `netlink` library to something else.